libarchive/entry.py

```diff
@@ -61,17 +61,36 @@ def isfifo(self):
 
     @property
     def islnk(self):
+        return bool(ffi.entry_hardlink_w(self._entry_p))
+
+    @property
+    def issym(self):
         return self.filetype & 0o170000 == 0o120000
```

Comment:
@Changaco nope, at least not if you want to keep the attribute names the same as in tarfile: 
islnk is for hardlinks and isym for symlinks